### DISCUSSION TITLE
Sharing a <connection> between services.

### DISCUSSION THREAD

**Discussion by farzbood** [dsoftwareinc/fakeredis-py#229] [Q&A] [answered]
Hi everyone,
I'm trying to test an **_event-driven_** **microservice architecture** solution, using **Redis pub/sub** as _**event-broker**_, locally.
Assumed that connecting to **fakeredis** with the same **_url_** from each microservice, shares that server between them (able to send and recieve event-messages through channels on shared fakeredis-server), but it doesn't work like that!
Is there a way to share a **fakeredis** **_connection/server_** between standalone microservices for test purpose?

Thanks in advance

**Comment by cunla** [comment, marked as answer]
fakeredis runs inside your process, and can't be shared between 2 processes.
